<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabDiveInformation</class>
 <widget class="QWidget" name="TabDiveInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KMessageWidget" name="multiDiveWarningMessage"/>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>403</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="diveInfoScrollAreaLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>4</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>

       <item row="0" column="0">
        <widget class="QLabel" name="diveHeadingLabel">
         <property name="text">
          <string>DIVE</string>
         </property>
         <property name="isHeader" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>

       <item row="1" column="0">
        <widget class="QGroupBox" name="diveModeBox">
         <property name="title">
          <string>Dive mode</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveModeLayout">
         <property name="alignment">
          <set>Qt::AlignBottom</set>
         </property>
          <item>
           <widget class="QComboBox" name="diveType">
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>Interval</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoSurfintervallLayout">
          <item>
           <widget class="QLabel" name="surfaceIntervalText">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string>Duration</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoDiveTimeLayout">
          <item>
           <widget class="QLabel" name="diveTimeText">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Max. depth</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoMaxDepthLayout">
          <item>
           <widget class="QLabel" name="maximumDepthText">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="1" column="4">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Avg. depth</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoAvgDepthLayout">
          <item>
           <widget class="QLabel" name="averageDepthText">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="2" column="0">
        <widget class="QLabel" name="gasHeadingLabel">
         <property name="text">
          <string>GAS</string>
         </property>
         <property name="isHeader" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>

       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Gas name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoGasesUsedLayout">
          <item>
           <widget class="QLabel" name="oxygenHeliumText">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Gas consumed</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoGasConsumedLayout">
          <item>
           <widget class="QLabel" name="gasUsedText">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>SAC</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoSacLayout">
          <item>
           <widget class="QLabel" name="sacText">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QGroupBox" name="groupBox_15">
         <property name="title">
          <string>CNS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoCnsLayout">
          <item>
           <widget class="QLabel" name="maxcnsText">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>OTU</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoOtuLayout">
          <item>
           <widget class="QLabel" name="otuText">
            <property name="text">
             <string/>
            </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="4" column="0">
        <widget class="QLabel" name="environmentHeadingLabel">
         <property name="text">
          <string>ENVIRONMENT</string>
         </property>
         <property name="isHeader" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>

       <item row="5" column="0">
        <widget class="QGroupBox" name="groupBox_airtemp">
         <property name="title">
          <string>Air temp.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoAirTempLayout">
          <item>
           <widget class="QLineEdit" name="airtemp">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="5" column="1">
        <widget class="QGroupBox" name="groupBox_watertemp">
         <property name="title">
          <string>Water temp.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoWaterTempLayout">
          <item>
           <widget class="QLineEdit" name="watertemp">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="5" column="2" colspan="3">
        <layout class="QGridLayout" name="envGroupLayout">
         <property name="horizontalSpacing">
          <number>4</number>
         </property>

       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_1">
         <property name="title">
          <string>Water type/Density</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="diveInfoWaterDensityLayout">
          <item>
           <widget class="QLabel" name="waterTypeText">
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="waterTypeCombo"/>
          </item>
          <item>
           <widget class="QLabel" name="salinityOverWrittenIcon">
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="salinityText">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string>Atm. pressure/Altitude</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
         <layout class="QHBoxLayout" name="diveInfoAirPressureLayout">
          <item>
           <widget class="QLineEdit" name="atmPressVal">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="atmPressType">
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       </layout>
       </item>

       <item row="6" column="0">
        <widget class="QGroupBox" name="groupBox_wavesize">
         <property name="title">
          <string>Surface waves</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property> 
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="surfaceWavesLayout">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="StarWidget" name="wavesize" native="true">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="waveLabel1">
            <property name="text">
             <string>Large</string>
            </property>
           </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignRight">
            <widget class="QLabel" name="waveLabel2">
             <property name="text">
              <string>Small</string>
             </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="6" column="1" alignment="Qt::AlignVCenter">
        <widget class="QGroupBox" name="groupBox_visibility">
         <property name="title">
          <string>Visibility</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="visibilityLayout">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="StarWidget" name="visibility" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="visLabel1">
            <property name="text">
             <string>Bad</string>
            </property>
           </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignRight">
            <widget class="QLabel" name="visLabel2">
             <property name="text">
              <string>Good</string>
             </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="6" column="2" alignment="Qt::AlignVCenter">
        <widget class="QGroupBox" name="groupBox_current">
         <property name="title">
          <string>Current</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
         <layout class="QGridLayout" name="currentLayout">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="StarWidget" name="current" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="curLabel1">
            <property name="text">
             <string>Strong</string>
            </property>
           </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignRight">
            <widget class="QLabel" name="curLabel2">
             <property name="text">
              <string>Weak</string>
             </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="6" column="3" alignment="Qt::AlignVCenter">
        <widget class="QGroupBox" name="groupBox_surge">
         <property name="title">
          <string>Surge</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
         <layout class="QGridLayout" name="surgeLayout">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="StarWidget" name="surge" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="surgeLabel1">
            <property name="text">
             <string>Strong</string>
            </property>
           </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignRight">
            <widget class="QLabel" name="surgeLabel2">
             <property name="text">
              <string>Weak</string>
             </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="6" column="4" alignment="Qt::AlignVCenter">
        <widget class="QGroupBox" name="groupBox_chill">
         <property name="title">
          <string>Chill</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter</set>
         </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
         <layout class="QGridLayout" name="cLayout">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
           <widget class="StarWidget" name="chill" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="chillLabel1">
            <property name="text">
             <string>Freezing</string>
            </property>
           </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignRight">
            <widget class="QLabel" name="chillLabel2">
             <property name="text">
              <string>Comfy</string>
             </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item row="7" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
